Installation/include/CGAL/gl.h

```diff
@@ -24,7 +24,11 @@
 #endif
 
 #ifdef __APPLE__
+#if TARGET_OS_IPHONE
+#  include <OpenGLES/ES2/gl.h>
+#else
 #  include <OpenGL/gl.h>
+#endif
 #else
 #  include <GL/gl.h>
 #endif
```

Comment:
Please indent the `if`, `include`, `else` and `endif` so that one can see easily the structure of the conditionals.
The code should be:
```C++
#ifdef __APPLE__
#  if TARGET_OS_IPHONE
#    include <OpenGLES/ES2/gl.h>
#  else
#    include <OpenGL/gl.h>
#  endif
#else
#  include <GL/gl.h>
#endif
```